parser-typechecker/src/Unison/Codebase/Path.hs

```diff
@@ -324,6 +326,9 @@ cons' n (Path' e) = case e of
 consAbsolute :: NameSegment -> Absolute -> Absolute
 consAbsolute n a = Absolute . cons n $ unabsolute a
 
+snocRelative :: Relative -> NameSegment -> Relative
```

Comment:
Could write a [Snoc](http://hackage.haskell.org/package/lens-4.19/docs/Control-Lens-Cons.html#t:Snoc) instance for all these similar cons/snoc things

Reply:
Interesting...

Reply:
I created #1200 to track this.